Agregados controladores de pago y listaDeTareas

matiasVarela076 opened the pull request:
- [x] Se agrego CPagoService, IPagoService,  PagoController
- [x] Se agrego CListaDeTareasService, IlistaDeTareasService, ListaDeTareasController 
- [x] Tiene los metodos crud basico, a medida que necesitemos algo especifico de cada clase se ira implementando
- [x] deje en pagoController una duda comentada en el getMapping de agregar pago. 
- [x] Faltaria los service y controlladora de calendario.
- [ ] Se agrego DetalleTareas model y Repository
- [ ] Se borro atributo link caso
- [ ] Se dejo por default description paso de 50 a 280 caracteres en la db
- [ ] se cambio el tamaño de varchar en el campo datoOtraParte

matiasVarela076 committed:
se mueve pagoService al paquete de implementacion

matiasVarela076 committed:
cosillas nuevas en service y controlladora de pago

matiasVarela076 committed:
Se agregan servicios y controladores de  pago y lista de tareas

nadialucila commented on src/main/java/com/proyectoIuris/iuris/service/implementacion/CUsuarioService.java:
```diff
@@ -20,6 +20,7 @@ public List<Usuario> list() {
         return usuarios;
     }
 
+    
```
👍

matiasVarela076 replied:
ta bien?

nadialucila commented on src/main/java/com/proyectoIuris/iuris/service/implementacion/CPagoService.java:
```diff
@@ -18,17 +19,25 @@ public List<Pago> getAll(){
     }
 
     @Override
-    public Optional<Pago> getPago(int idPago){
```
Yo tuve que cambiar en algunas ocasiones el Optional porque despues no podia pasar los datos a la vista, fijate despues si te da problemas

nadialucila commented on src/main/java/com/proyectoIuris/iuris/service/implementacion/CListaDeTareasService.java:
```diff
@@ -0,0 +1,36 @@
+            listaDeTareasRepository.save(listToDo);
+            return true;
+        } else return false;
+    } //El save tengo entendido que si no existe lo crea y sino se edita (pero creo que habia fallado, despues revisamos)
```
Funciona asi, lo dice en la documentación

matiasVarela076 replied:
oki entonces update no lo metemo por ahora

matiasVarela076 committed:
Merge branch 'developer' into developer_Iuris_Matt

matiasVarela076 committed:
se agrega modelo DetalleTarea y algunos cambios mas

matiasVarela076 committed:
Merge branch 'developer_Iuris_Matt' of https://github.com/pablofiamingo/iuris into developer_Iuris_Matt

matiasVarela076 committed:
cambios

matiasVarela076 committed:
- se crea paquete interfaces para mover todas las interfaces de los servicios
- se borro link, y cambio tamaño varchar de descripcion y datoOtraParte en el model/Caso
